addons/sale/models/product_product.py

```diff
@@ -17,12 +17,10 @@ def _compute_sales_count(self):
             return r
         date_from = fields.Datetime.to_string(fields.datetime.combine(fields.datetime.now() - timedelta(days=365),
                                                                       time.min))
-        date_to = fields.Datetime.to_string(fields.datetime.combine(fields.datetime.today(), time.min))
         domain = [
-            ('state', 'in', ['sale', 'done', 'paid']),
+            ('state', 'in', ['sale', 'done', 'paid', 'pos_done', 'invoiced']),
```

Comment:
`pos_done` and `invoiced` are not present here https://github.com/odoo/odoo/blob/master/addons/sale/report/sale_report.py#L40

but here https://github.com/odoo/odoo/blob/master/addons/pos_sale/report/sale_report.py#L11

I think it is better to inherit `_compute_sales_count` in `pos_sale/report/sale_report.py` and not in `sale/report/sale_report.py`